stubs/redis/redis/sentinel.pyi

```diff
@@ -39,5 +42,11 @@ class Sentinel(object):
     def discover_master(self, service_name): ...
     def filter_slaves(self, slaves): ...
     def discover_slaves(self, service_name): ...
-    def master_for(self, service_name, redis_class=..., connection_pool_class=..., **kwargs) -> None: ...
-    def slave_for(self, service_name, redis_class=..., connection_pool_class=..., **kwargs) -> None: ...
+    @overload
+    def master_for(self, service_name: str, *, connection_pool_class=..., **kwargs) -> Redis[Any]: ...
```

Comment:
```suggestion
    def master_for(self, service_name: str, *, connection_pool_class=..., **kwargs) -> Redis[str]: ...
```

Reply:
Why str, it can also be bytes...
Or should I have another override?

Reply:
You are right. For some reasons I assumed the redis instance was constructed using default arguments. But it can be bytes as well, so `Any` is fine.